ckanext/harvest/queue.py

```diff
@@ -211,8 +211,14 @@ def __init__(self, redis, routing_key):
     def consume(self, queue):
         while True:
             key, body = self.redis.blpop(self.routing_key)
-            self.redis.set(self.persistance_key(body),
-                           str(datetime.datetime.now()))
+            try:
+                self.redis.set(self.persistance_key(body),
+                            str(datetime.datetime.now()))
+            except Exception:
```

Comment:
We can just have `except:` here since we don't care the type of exception.

Reply:
👍

Reply:
We decided to remove the stack trace as the exception is raised as an indicator that the redis data is corrupt.